packages/contract/README.md

```diff
@@ -80,51 +80,47 @@ v4.0.2 or higher.
 Let's use `@truffle/contract` with an example contract from [Dapps For Beginners](https://dappsforbeginners.wordpress.com/tutorials/your-first-dapp/). In this case, the abstraction has been saved to a `.sol` file by [@truffle/artifactor](https://github.com/trufflesuite/truffle/tree/master/packages/artifactor):
 
 ```javascript
+const contract = require("@truffle/contract");
 // Require the package that was previosly saved by @truffle/artifactor
-var MetaCoin = require("./path/to/MetaCoin.json");
+const MetacoinArtifact = require("./path/to/MetaCoin.json");
+const instance = contract(MetacoinArtifact);
```

Comment:
I might be mistaken, but won't using `contract` like this get you something for the class, not the instance?

Reply:
ohh good catch, I'll fix, thanks!